File: static/UserView/carrito.js
function eliminarDelCarrito(idArticulo) {
    fetch('/eliminar_carrito', {
        method: 'POST',
        headers: {
            'Content-Type': 'application/json'
        },
        body: JSON.stringify({ id_articulo: idArticulo })
    })
    .then(response => response.json())
    .then(data => {
        if(data.success) {
            location.reload();
        } else {
            alert('Error al eliminar del carrito');
        }
    });
}

File: Model/carrito.py
from flask import session, jsonify

def agregar_carrito_func(data):
    id_articulo = str(data.get('id_articulo'))
    cantidad = int(data.get('cantidad', 1))
    carrito = session.get('carrito', {})
    if id_articulo in carrito:
        carrito[id_articulo] += cantidad
    else:
        carrito[id_articulo] = cantidad
    session['carrito'] = carrito
    session.modified = True
    return jsonify({'success': True})
